afs/afs-core/src/main/java/com/powsybl/afs/ProjectFile.java

```diff
@@ -22,7 +22,7 @@
  */
 public class ProjectFile extends ProjectNode {
 
-    private final WeakListenerList<ProjectFileListener> listeners = new WeakListenerList<>();
+    protected final WeakListenerList<ProjectFileListener> listeners = new WeakListenerList<>();
```

Comment:
Why do you need to increase the visibility of the listeners? Could you use `addListener` and `removeListener` instead?
Maybe you need to add a new method to ProjectFile to notify listeners?